README.md

```diff
@@ -96,13 +96,35 @@ for s, p, o in np.provenance:
 print(np.introduces_concept)
 ```
 
-### Specifying more information
-You can optionally specify that the ```Publication``` introduces a particular concept, or is derived from another nanopublication:
+### Specifying concepts relative to the nanopublication namespace
+You can optionally specify that the ```Publication``` introduces a particular concept using blank nodes. 
+The pubinfo graph will note that this nanopub npx:introduces the concept. The concept should be a blank node 
+(rdflib.term.BNode), and is converted to a URI derived from the nanopub's URI with a fragment (#) made from the blank
+node's name.
 ```python
+from rdflib.term import BNode
 publication = Publication.from_assertion(assertion_rdf=my_assertion,
-                                         introduces_concept=(URIRef('www.example.org/timbernerslee'),
-                                         derived_from=URIRef('www.example.org/another-nanopublication') )
+                                         introduces_concept=BNode('timbernerslee'))
 ```
+Upon publication, any blank nodes in the rdf graph are replaced with the nanopub's URI, with the blank node name as a
+fragment. For example, if the blank node is called 'step', that would result in a URI composed of the nanopub's (base)
+URI, followed by #step. In case you are basing your publication on rdf that has a lot of concepts specific to this 
+nanopublication that are not blank nodes you could use `replace_in_rdf` to easily replace them with blank nodes:
+```python
+from nanopub import replace_in_rdf
+replace_in_rdf(rdf=my_assertion, oldvalue=URIRef('www.example.org/timbernerslee'), newvalue=BNode('timbernerslee'))
```

Comment:
This is fine, but I'm wondering if we should instead return a new rdf graph instead of modifying this one in-place? I suppose users might not always want to modify their original graph. Then again, this is pretty minor, so we should just keep it like this unless/until a specific use case shows up. The documentation in the README looks good to me though.

Reply:
Yeah, that's a good point. I am neutral. The user can always do a copy themselves. So I will keep it like this for now.